tests/test_modular_domains/test_modular_domains.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+import openmoc
+from testing_harness import TestHarness
+from input_set import SimpleLatticeInput
+
+
+class ModularDomainsTestHarness(TestHarness):
+    """An eigenvalue calculation in a 3D lattice with 7-group C5G7 data, cut in 
+    2*3*4 modular domains. Modular domains are meant to replicate domain 
```

Comment:
The modular domains will only replicate the track laydown. The final results have tiny differences since the sweeping process are different.